### PR TITLE
Polish Chinese translation

### PR DESCRIPTION
**Pull request by liyishuai**
Localise with English annotated

Less delocalization than #539
@jalenchuh Please review

### CODE REVIEW

**Review by jalenchuh — APPROVED** [review]

**Comment by anuraghazra** [comment]
hi @liyishuai can you update the snapshots? tests are failing.

**Comment by anuraghazra** [comment]
Sorry about that the tests aren't really nicely written and needs some refactoring #705

File: tests/renderStatsCard.test.js [hunk 1]
```diff
@@ -240,26 +240,26 @@ describe("Test renderStatsCard", () => {
       document.querySelector(
         'g[transform="translate(0, 0)"]>.stagger>.stat.bold',
       ).textContent,
-    ).toBe("总 Star:");
+    ).toBe("获标星（star）:");
```
Comment (liyishuai) [inline tests/renderStatsCard.test.js:243]: #775

File: tests/renderStatsCard.test.js [hunk 2]
```diff
@@ -240,26 +240,26 @@ describe("Test renderStatsCard", () => {
...
     expect(
       document.querySelector(
         'g[transform="translate(0, 25)"]>.stagger>.stat.bold',
       ).textContent,
-    ).toBe(`总提交 (${new Date().getFullYear()}):`);
+    ).toBe(`累计提交（commit） (${new Date().getFullYear()}):`);
```
Comment (liyishuai) [inline tests/renderStatsCard.test.js:248]: This would render `累计提交（commit） (2021)`, which could be improved beyond this PR.

Comment (anuraghazra) [reply tests/renderStatsCard.test.js:248]: Yes will do something about that later on.

**Comment by anuraghazra** [comment]
Thanks @liyishuai :)

**Pull request merged**